src/js/background/context-menu.js

```diff
@@ -30,6 +30,13 @@ const staticMenuData = {
     visible: true,
     contexts: ["all"],
   },
+  insertPhoneMask: {
+    id: "fx-private-relay-insert-phone-mask",
+    title: browser.i18n.getMessage("pageInputIconInsertPhoneMask"),
+    enabled: true,
```

Comment:
thought (non-blocking): Not sure if we want to show this greyed out / disabled when the user does not have a phone mask set. If so, you'll want to follow the pattern like in `generateAliasDisabled` where you make another context menu entry like `insertPhoneMask`, but set `enabled: false`.

Reply:
Thanks for the pointers. I think for now we're good, since the barrier to being able to insert a phone mask is quite a bit higher than the one to being able to generate a mask, in addition to there not being a clear path for users on how to enable it, and it not being listed in the acceptance criteria.

Reply:
Yeah - I know this was stripped-down MVP feature set. I figure there will be some sort of upsell surface(s) in the future. Good to go for now.